`SmallVec::reserve_one_unchecked` uses the `rust-cold` calling convention.

diff -ru a/smallvec-1.15.1/src/lib.rs b/smallvec-1.15.1/src/lib.rs
--- a/smallvec-1.15.1/src/lib.rs
+++ b/smallvec-1.15.1/src/lib.rs
@@ -89,6 +89,7 @@
 //!
 //! Tracking issue: [rust-lang/rust#34761](https://github.com/rust-lang/rust/issues/34761)
 
+#![feature(rust_cold_cc)]
 #![no_std]
 #![cfg_attr(docsrs, feature(doc_cfg))]
 #![cfg_attr(feature = "specialization", allow(incomplete_features))]
@@ -1226,7 +1227,7 @@
 
     /// Internal method used to grow in push() and insert(), where we know already we have to grow.
     #[cold]
-    fn reserve_one_unchecked(&mut self) {
+    extern "rust-cold" fn reserve_one_unchecked(&mut self) {
         debug_assert_eq!(self.len(), self.capacity());
         let new_cap = self.len()
             .checked_add(1)
